homeassistant/components/recorder/__init__.py

```diff
@@ -579,3 +587,19 @@ def _close_run(self):
             self.event_session.close()
 
         self.run_info = None
+
+
+def _remove_display_attributes_from_state(state):
+    """Remove attributes that are primarily used for the display layer.
+
+    These attributes are mostly static and
+    take of a lot of space in the database
+    and are not needed for any core functionality.
+    """
+    if not any(k in state.attributes for k in DISPLAY_ATTRIBUTES):
```

Comment:
if `DISPLAY_ATTRIBUTES` was a `set()` then you could do:
```
if not DISPLAY_ATTRIBUTES.intersection(state.attributes):
    return
```
🤷

Reply:
done